### ISSUE TITLE
Post search results to channel

### ISSUE THREAD

**Issue by jorilallo** [outline/outline#613]
Right now `/outline <search term>` results are only visible to the user performing the action. It might be neat to add Slack actions to post individual search results into the channel.

This would be possible after #603

**Comment by jacobsmith** [comment]
I would like to add that this would be super helpful. I'm bringing on a junior developer this summer and being able to see what things he is searching for will be helpful for me to know what additional content to create.

I'm familiar with JavaScript and React, though I haven't done much server-side JS - if I get time, I'll try and write a PR to add this functionality. I'd also love to get a "preview" of the document (like the 20 words on either side of the found search term) show up in Slack, as that might be enough to solve the issue. (I know this isn't a feature request forum, I'm just thinking out loud of things I'd like to see implemented or implement myself (: ).

**Comment by tommoor** [comment]
> I'm familiar with JavaScript and React, though I haven't done much server-side JS - if I get time, I'll try and write a PR to add this functionality

Sounds great, let me know if you have any issues or questions. Headsup that the Slack functionality can be tricky to work on as you need to register a client and setup the webhooks to go to your local machine.

> I'd also love to get a "preview" of the document (like the 20 words on either side of the found search term)

Maybe we should knock in a separate issue for this?

**Comment by tommoor** [comment]
As of 61b6b1814829f99e3d6630c1b93cf941d2330877 we're now showing the snippet related to the search term in results from the `/outline` slash command in Slack 😄
